src/openscad.cc

```diff
@@ -784,6 +784,16 @@ int gui(const vector<string> &inputFiles, const fs::path &original_path, int arg
 }
 #endif // OPENSCAD_QTGUI
 
+std::pair<string, string> customSyntax(const string& s)
+{
+#if defined(Q_OS_MACX)
+	if (s.find("-psn_") == 0)
+		return make_pair(string("psn"), s.substr(5));
+#endif
+
+  return make_pair(string(), string());
```

Comment:
Tiny suggestion: `return {};` should do the job here.

Reply:
Thanks for your insight.